Src/Plugins/ValidatorBase.cs

```diff
@@ -56,6 +56,18 @@ public static string CreateReturnValueForCompromisedAsset(string asset, string u
                 nameof(ValidationState.Authorized) + $"#The '{user}' account is compromised for '{asset}'.";
         }
 
+        public static string CreateReturnValueForFileException(Exception e, string asset)
```

Comment:
>CreateReturnValueForFileException [](start = 29, length = 33)

This is not a good helper, because it knows about a very specific exception message. That kind of logic should always be in a rule. i.e., you should have a handler that catches the specific kind of exception that has this message.

Reply:
reverted

---
In reply to: [554483316](https://github.com/microsoft/sarif-pattern-matcher/pull/96#discussion_r554483316) [](ancestors = 554483316)